cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -1129,4 +1131,36 @@
     public static final String EXTENSIONS_DIR = "operational.stats.extensions.dir";
     public static final String REFRESH_INTERVAL_SECS = "operational.stats.refresh.interval.secs";
   }
+
+  /**
+   * Constants for retry policies.
+   */
+  public static final class Retry {
+    private static final String PREFIX = "retry.policy.";
+    public static final String TYPE = PREFIX + "type";
+    public static final String MAX_TIME_SECS = PREFIX + "max.time.secs";
+    public static final String MAX_RETRIES = PREFIX + "max.retries";
+    public static final String DELAY_BASE_MS = PREFIX + "delay.base.ms";
+    public static final String DELAY_MAX_MS = PREFIX + "delay.max.ms";
+
+    public static String type(ProgramType programType) {
```

Comment:
As I mentioned above, I am not sure if it good style to have methods in a Constants class. Not a biggie.